test/metamodelutil.js

```diff
@@ -100,6 +100,95 @@ describe('MetaModel (Car)', () => {
     });
 });
 
+describe('MetaModel  aliasing', () => {
+
+    it('should convert a CTO model to its metamodel with name resolution', async () => {
+        const ModelPath = path.resolve(__dirname, './cto/aliasedImport.json');
+        const Model = JSON.parse(fs.readFileSync(ModelPath, 'utf8'));
+        const MetaModelResolved = JSON.parse(fs.readFileSync(path.resolve(__dirname, './cto/aliasedImportResolved.json'), 'utf8'));
+
+        const mm1r = MetaModelUtil.resolveLocalNamesForAll(Model);
+        mm1r.should.deep.equal(MetaModelResolved);
+    });
+
+    it('Should throw if name not found',async()=>{
```

Comment:
nit: description of this test could be a bit more clear. 
```suggestion
    it("Should throw if name not found because it's aliased",async()=>{
```

Reply:
It is based on the eslint.yml rules at the root of the repo. If we use double quotes, it will throw an eslint error.